packages/sitecore-jss-angular/package.json

```diff
@@ -35,7 +35,7 @@
     "@angular/platform-browser-dynamic": "~15.2.9",
     "@angular/router": "~15.2.9",
     "@types/jasmine": "^3.4.1",
-    "@types/node": "^14.14.35",
+    "@types/node": "^16.11.6",
```

Comment:
This version should be bumped to 18, and it's already applied in _dev_: https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss-angular/package.json#L38
You can pull latest changes from _dev_